examples/todo-jwt/src/models/todo.model.ts

```diff
@@ -30,12 +30,6 @@ export class Todo extends Entity {
   })
   isComplete?: boolean;
 
-  @property({
```

Comment:
I removed this tag property. It is not in the original todo example documentation, but in that completed example, it is added and used in a mocha test.  For this sample, I am removing it as there is no test trying to access it.